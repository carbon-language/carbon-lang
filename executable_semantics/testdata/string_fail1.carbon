// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
// Exceptions. See /LICENSE for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
//
// RUN: not executable_semantics %s 2>&1 | \
// RUN:   FileCheck --match-full-lines --allow-unused-prefixes=false %s
// RUN: not executable_semantics --trace %s 2>&1 | \
// RUN:   FileCheck --match-full-lines --allow-unused-prefixes %s
// AUTOUPDATE: executable_semantics %s
// CHECK: COMPILATION ERROR: {{.*}}/string_fail1.carbon:16: Invalid escaping in string: "str\e"


package ExecutableSemanticsTest api;

fn main() -> i32 {
  Print("str\e");
  return 0;
}
