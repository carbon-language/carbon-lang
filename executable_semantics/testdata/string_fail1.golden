COMPILATION ERROR: 6: Invalid escaping in string: "str\e"
EXIT CODE: 255
